test/e2e/kubevirt.go

```diff
@@ -2359,4 +2360,97 @@ chpasswd: { expire: False }
 		)
 	})
 
+	Context("MAC conflicts detection", func() {
+		newTestVMI := func(name, mac string) *kubevirtv1.VirtualMachineInstance {
```

Comment:
let's hint at this being parametrized by the MAC. `newTestVMIWithMACRequest` ? Plus, in which interface will this MAC be set ? does it make sense to have that hardcoded ?